### PR TITLE
PostMVPified (and/or just deleted) some tests

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

**Comment by cjllanwarne** [comment]
Sorry @scottfrazer and @mcovarr, you're the only people here to review this! It should be very quick though!

**Comment by mcovarr** [comment]
yes @scottfrazer and I have distinguished ourselves by our non-absence.  😄

File: engine/src/test/scala/cromwell/WorkflowOutputsSpec.scala [hunk 1]
```diff
@@ -59,17 +59,5 @@ class WorkflowOutputsSpec extends CromwellTestkitSpec {
...
-      runWdlAndAssertOutputs(
-        sampleWdl = SampleWdl.ThreeStepWithInputsInTheOutputsSection,
-        eventFilter = EventFilter.info(pattern = s"is in a terminal state: WorkflowSucceededState", occurrences = 1),
-        runtime = "",
-        expectedOutputs = Map(
-          "three_step.cgrep.pattern" -> AnyValueIsFine
-        ),
-        allowOtherOutputs = false
-      )
-    }
```
Comment (mcovarr) [inline engine/src/test/scala/cromwell/WorkflowOutputsSpec.scala]: I have a nagging feeling we supported this for a legit reason but I just don't remember what it was...

**Comment by mcovarr** [comment]
:+1:

File: engine/src/test/scala/cromwell/WorkflowOutputsSpec.scala (hunk 1 above)
Comment (cjllanwarne) [inline engine/src/test/scala/cromwell/WorkflowOutputsSpec.scala]: I can sort of imagine a future world where workflows are composable, and you want to just "forward" an input through to the next stage in a pipeline of workflows... But for now it doesn't really serve any purpose

**Comment by scottfrazer** [comment]
👍

**Pull request merged**